index.html

```diff
@@ -2943,6 +2944,229 @@ <h5><code>unsubscribeallevents</code></h5>
+          multiple Things. Consumers implement a Webhook listener that is handling the event 
+          streams generated by Things. 
+          Depending on the use case, a single listener for multiple things and multiple 
+          event types may be used.
+        </p>
+        <section id="http-webhook-profile-protocol-binding-subscribeevent">
+          <h5><code>subscribeevent</code></h5>
+          <p class="rfc2119-assertion" 
+            id="http-baseline-profile-protocol-binding-subscribeevent-1">
+            The URL of an <code>Event</code> resource to be used when 
+            subscribing to an event MUST be obtained from a Thing Description 
+            by locating a 
+            <a href="https://www.w3.org/TR/wot-thing-description/#form">
+            <code>Form</code></a> inside the corresponding 
+            <a href="https://www.w3.org/TR/wot-thing-description/#eventaffordance">
+            <code>EventAffordance</code></a> for which:
+          </p>
+          <ul class="rfc2119-assertion" 
+            id="http-webhook-profile-protocol-binding-subscribeevent-2">
+            <li>
+              After defaults have been applied, its <code>op</code> member 
+              contains the value <code>subscribeevent</code>
+            </li>
+            <li>
+              After being resolved against a base URL where applicable, the
+              <a href="https://tools.ietf.org/html/rfc3986#section-3.1">URI
+              scheme</a> [[RFC3986]] of the value of its <code>href</code>
+              member is <code>http</code> or <code>https</code>
+            </li>
+            <li>Its <code>subprotocol</code> member has a value of
+              <code>"webhook"</code>
+            </li>
+          </ul>
+          <p class="rfc2119-assertion" 
+            id="http-webhook-profile-protocol-binding-subscribeevent-3">
+            The resolved value of the <code>href</code> member MUST then be used
+            as the URL of the <code>Event</code> resource.
+          </p>
+          <p class="rfc2119-assertion" 
+            id="http-webhook-profile-protocol-binding-subscribeevent-4">
+            In order to subscribe to an event, a Consumer MUST  
+            provide the listener URL in the data payload of the 
+            subscribe operation of the <code>Event</code> resource.
+          </p>
+          <p>
+            This involves the Consumer sending an HTTP request to
+            the Web Thing with:
+          </p>
+          <ul>
+            <li>Method set to <code>POST</code></li>
+            <li>URL set to the URL of the <code>Event</code> resource</li>
+            <li>
+              <code>Accept</code> header set to <code>application/json</code>
+            </li>
+            <li>
+              <code>Request Payload</code> contains a <code>JSON</code> 
+              object with a subscription payload.
+            </li>
+
+            <!-- FIXME: define subscription payload -->
```

Comment:
Please fix this or turn it into an Editor's Note so that it isn't missed.

Note that the content of this payload should match the schema in the subscription member in the example above.

Reply:
turned into ednote